src/megameklab/com/util/UnitUtil.java

```diff
@@ -959,21 +930,22 @@ public static boolean isJumpJet(Mounted m) {
                 || m.getType().hasFlag(MiscType.F_JUMP_BOOSTER);
     }
 
-    public static String getJumpJetType(int type, boolean clan) {
+    /**
+     *
+     * @param type The value returned by {@link Mech#getJumpType()}
+     * @return     The {@link EquipmentType} lookup key for the jump jet
+     */
+    public static String getJumpJetType(int type) {
         if (type == Mech.JUMP_IMPROVED) {
-            if (clan) {
-                return "CLImprovedJump Jet";
-            } else {
-                return "ISImprovedJump Jet";
-            }
+            return EquipmentTypeLookup.IMPROVED_JUMP_JET;
```

Comment:
IS and Clan IJJs used to be different equipment because they have a different intro date, but they have been merged since the IO tech advancement update since they are functionally identical.